### DISCUSSION TITLE
Enable nvim cmp to be on by default

### DISCUSSION THREAD

**Discussion by InSynchronous** [VonHeikemen/lsp-zero.nvim#242] [Q&A]
Hi I'm using lsp-zero on its recommended config on version 2.0 and want autocomplete suggestions to be on by default(like vscode).

**Comment by VonHeikemen** [comment]
If you do have the "recommended" config, nvim-cmp should be on.

**Comment by InSynchronous** [comment]
Nvm I was using a snip-it of code from someone who was doing it manually. Fixed:
local lsp = require('lsp-zero').preset("recommended")

lsp.setup()
